src/main/io/osd.c

```diff
@@ -197,7 +197,8 @@ static const uint8_t osdElementDisplayOrder[] = {
     OSD_NUMERICAL_HEADING,
     OSD_NUMERICAL_VARIO,
     OSD_COMPASS_BAR,
-    OSD_ANTI_GRAVITY
+    OSD_ANTI_GRAVITY,
+    OSD_LOG_STATUS
```

Comment:
Please put a comma after the last entry - this way, when another entry is added, the previous line will not show up in the diff, making reviewing easier.